packages/hdwallet-keepkey/src/keepkey.ts

```diff
@@ -984,6 +985,9 @@ export class KeepKeyHDWallet implements core.HDWallet, core.BTCWallet, core.ETHW
     this._supportsEos = semver.gte(fwVersion, "v6.4.0");
     // this._supportsThorchain = Semver.get(fwVersion, "v7.0.0");
 
+    // EIP-1559 isn't supported until v7.2.1
+    this._supportsEip1559 = semver.gte(fwVersion, "v7.2.1");
+
```

Comment:
👍 
didn't realize we already had this pattern

Reply:
we should probably do this for other things like `_supportsThorChain` in another PR